code/src/java/pcgen/system/Main.java

```diff
@@ -129,6 +134,19 @@ private static void logSystemProps()
 	}
 
 	/**
+	 * Don't delete. Used for testing logging.
```

Comment:
this is mainly there so I could add a line to main() and run. It was very useful for testing that the classpath was set up correctly. This differs from main to test. I don't /really/ need this function. It was just nice to have.